packages/grid/_modules_/grid/components/ScrollArea.tsx

```diff
@@ -2,75 +2,79 @@ import * as React from 'react';
 import { COL_REORDER_START, COL_REORDER_STOP, SCROLLING } from '../constants/eventsConstants';
 import { ScrollParams, useApiEventHandler } from '../hooks';
 import { ApiRef } from '../models';
+import { classnames } from '../utils';
 import { ApiContext } from './api-context';
 
-enum Direction {
-  Left = 'left',
-  Right = 'right',
-}
-
-type ScrollDirection = 'left' | 'right';
-
 interface ScrollAreaProps {
-  scrollDirection: ScrollDirection;
+  scrollDirection: 'left' | 'right';
 }
-export const ScrollArea: React.FC<ScrollAreaProps> = React.memo(
-  ({ scrollDirection }: ScrollAreaProps) => {
-    const api = React.useContext(ApiContext);
-    const cursorPossitionX = React.useRef<number | null>(null);
-    const cssClass = `MuiDataGrid-scrollArea ${
-      scrollDirection === Direction.Left
-        ? 'MuiDataGrid-scrollArea-left'
-        : 'MuiDataGrid-scrollArea-right'
-    }`;
-    const [isDragging, setIsDragging] = React.useState<boolean>(false);
-    const scrollStep = React.useRef<number>(2);
-    const scrollStepMultiplier = 3;
-    const currentScrollPosition = React.useRef<ScrollParams | null>({
-      left: 0,
-      top: 0,
-    });
 
-    const setCurrentScrollPosition = React.useCallback(
-      (scrollPosition) => {
-        currentScrollPosition.current = scrollPosition;
-      },
-      [currentScrollPosition],
```

Comment:
refs shouldn't be dependencies, they never change during a mount lifecycle

Reply:
Thanks, I won't do that mistake again